.travis.yml

```diff
@@ -1,16 +1,63 @@
 language: python
 python:
   - "2.7"
-
+ 
+addons:
+  postgresql: "9.3"  
+  # https://github.com/travis-ci/travis-ci/issues/6972
+  apt:
+    packages:
+    - postgresql-9.3-postgis-2.3
+ 
 env:
   global:
+    # Work around boto issue https://github.com/travis-ci/travis-ci/issues/5246#issuecomment-166460882 , https://github.com/tendenci/tendenci/issues/539#issuecomment-345092987
+    - BOTO_CONFIG=/doesnotexist
     # for $TENDENCI_TRANSIFEX_PASSWORD
-    secure: "H3pfvt1czYywa1Fp+quLeWqHOTqZVXDeyoCAFCthi4FO7YYvj8FpaRefUJ8iBIpN/NKImJkGGAh10KWkWXwhwWf2IZQmjTpYRZ0EltkhLyHDIQugoKCnt6VSfPUcrBwZBbA5mBI1JWtSWuN6MnrwMbM9XPpYf4o85OpZZnBdCN8="
+    - secure: "H3pfvt1czYywa1Fp+quLeWqHOTqZVXDeyoCAFCthi4FO7YYvj8FpaRefUJ8iBIpN/NKImJkGGAh10KWkWXwhwWf2IZQmjTpYRZ0EltkhLyHDIQugoKCnt6VSfPUcrBwZBbA5mBI1JWtSWuN6MnrwMbM9XPpYf4o85OpZZnBdCN8="
+
+before_install:
+ - sudo apt-get -qq update
+ - sudo apt-get -qq install build-essential python-dev libevent-dev libpq-dev libjpeg8 libjpeg-dev libfreetype6 libfreetype6-dev
+ - sudo apt-get -qq install git
+ - sudo apt-get -qq install python-pip
 
 # command to install dependencies
 install:
-  - pip install "Django>=1.8,<1.9"
+ - cd ../..
+ - pwd
+ - pip install -q "Django>=1.8,<1.9"
+ - pip install -e goetzk/tendenci
```

Comment:
Why `pip install -e goetzk/tendenci`? Is "goetzk/tendenci" your github repo?

Reply:
Hi @jennyq ,
It is my GitHub repo, and Travis clones to the name of the repository. I've fixed that in 815a658 where I swap to using a Travis supplied variable.

The current branch is running on Travis at the moment.
https://travis-ci.org/goetzk/tendenci/builds/399389030